theano/gof/fg.py

```diff
@@ -763,11 +759,8 @@ def check_integrity(self):
                         "Inconsistent clients list.", variable, node.inputs[i]
                     )
 
-    def __str__(self):
-        return "[%s]" % ", ".join(graph.as_string(self.inputs, self.outputs))
-
     def __repr__(self):
-        return self.__str__()
+        return f"FunctionGraph({', '.join(graph.as_string(self.inputs, self.outputs))})"
```

Comment:
I'm not opposed, just curious, Why change the string repr to include the substring `FunctionGraph`. Just to make it more explicit on the terminal?

Reply:
Yeah, its `repr` doesn't even say what it is, which is especially annoying when it's printed from within another objects `str` output.